frontend/src/bundles/friends/components/friend-details/friend-details.tsx

```diff
@@ -0,0 +1,92 @@
+import { UserCircleIcon } from '@heroicons/react/24/solid';
+
+import { actions as achievementsActions } from '~/bundles/achievements/store/achievements.js';
+import { Avatar, Card } from '~/bundles/common/components/components.js';
+import { ViewAllButton } from '~/bundles/common/components/info-section/components/view-all-button/view-all-button.js';
+import { AppRoute } from '~/bundles/common/enums/enums.js';
+import { formatDateString } from '~/bundles/common/helpers/format-date-string/format-date-string.helper.js';
+import { validateImageUrl } from '~/bundles/common/helpers/validate-image-url/validate-image-url.helper.js';
+import {
+    useAppDispatch,
+    useAppSelector,
+    useEffect,
+} from '~/bundles/common/hooks/hooks.js';
+
+type Properties = {
+    id: number;
+    name: string | null;
+    isActive: boolean;
+    avatarUrl: string | null;
+};
+
+const FriendDetails = ({
+    id,
+    name,
+    isActive,
+    avatarUrl,
+}: Properties): JSX.Element => {
+    const dispatch = useAppDispatch();
+
+    const { achievements } = useAppSelector(({ achievements }) => ({
+        achievements: achievements.achievements,
+    }));
+
+    useEffect(() => {
+        void dispatch(achievementsActions.getAchievementsByUserId(id));
+    }, [dispatch, id]);
+
+    return (
+        <div className={'flex flex-col items-center justify-center '}>
+            <div className="mb-4 flex items-center gap-2">
+                {isActive ? (
+                    <div className="bg-buttonPrimary h-2 w-2 rounded-[50%]" />
+                ) : (
+                    <div className="bg-buttonTertiary h-2 w-2 rounded-[50%]" />
+                )}
```

Comment:
Just use getValidClassNames to avoid repetitive code.

Reply:
I changed it somewhat differently:

```
<div className={`${isActive ? 'bg-buttonPrimary' : 'bg-buttonTertiary'} bg-buttonPrimary h-2 w-2 rounded-[50%]`} />
```

Reply:
Actually, you still need to use getValidClassNames though

Reply:
@ArtemLymarenko @SofiiaTrokhymchuk Yes, thank you. I fixed it using getValidClassNames